src/Controllers/ModuleSuiteLocator.php

```diff
@@ -190,4 +196,30 @@ protected function loadSuiteConfiguration($suite, Module $module)
             'settings' => $resolvedConfig,
         ];
     }
+
+    private function setI18nLocale(array $config): void
+    {
+        $path = $config['type']['settings']['paths'][0] ?? '';
+        $packagistName = '';
+        if (preg_match('#^vendor/([^/]+/[^/]+)/.+#', $path, $matches)) {
+            // Running behat tests of a module in the vendor folder
+            $packagistName = $matches[1];
+        } else {
+            // Running behat tests of a module or project in the root folder
+            $file = BASE_PATH . '/composer.json';
```

Comment:
```suggestion
            $file = Path::join(BASE_PATH, 'composer.json');
```
People might be running a WAMP stack so need windows dir separators.

Reply:
Updated